src/components/InviteMemberProject/index.js

```diff
@@ -148,21 +179,41 @@ const InvaiteMemberProject = () => {
                 <TableCell>User</TableCell>
                 <TableCell align="right">Last active</TableCell>
                 <TableCell align="right">Status</TableCell>
-                <TableCell align="right">Ations</TableCell>
+                <TableCell align="right">Actions</TableCell>
               </TableRow>
             </TableHead>
             <TableBody>
-              {rows.map((row) => (
+              {users.map((user) => (
```

Comment:
Add a check here. What happen if there is no `users` or `undefined`?

Reply:
solved